adafruit_stmpe610.py

```diff
@@ -26,12 +26,21 @@
 import time
 from micropython import const
 
+try:
+    from typing import List, Optional, Tuple, Union
+    from typing_extensions import Literal
+    from microcontroller import Pin
+    from busio import I2C, SPI
+except ImportError:
+    pass
 
 __version__ = "0.0.0+auto.0"
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_STMPE610.git"
 
 
-def map_range(x, in_min, in_max, out_min, out_max):
+def map_range(
+    x: Union[int, float], in_min: int, in_max: int, out_min: int, out_max: int
```

Comment:
When typing something as a `float` it is assumed `int` may also be used.  Essentially, `float` can be used in place of `Union[int, float]`.  See [PEP 484](https://peps.python.org/pep-0484/#the-numeric-tower) for more information.